box.json

```diff
@@ -0,0 +1,42 @@
+{
+  "finder": [
+    {
+      "in": "src"
+    },
+    {
+      "name": "autoload.php",
+      "in": "vendor"
+    },
+    {
+      "name": [
+        "autoload_real.php",
+        "ClassLoader.php",
+        "autoload_static.php"
+      ],
+      "in": [
+        "vendor/composer"
+      ]
+    },
+    {
+      "name": [
+        "*.php",
+        "LICENSE"
+      ],
+      "exclude": [
+        "Tests",
+        "Tester"
+      ],
+      "in": [
+        "vendor/symfony/console",
+        "vendor/symfony/filesystem",
+        "vendor/symfony/process",
+        "vendor/symfony/stopwatch",
+        "vendor/symfony/polyfill-mbstring"
```

Comment:
Hm, not so simple.
It will break the global "Test" exclusions, and force you to define it for each vendor. That's how finder works...
But maybe there's a trick with finder, i don't know.
Btw, this is how composer does.